monokai-theme.el

```diff
@@ -5652,6 +5652,44 @@ Also affects 'linum-mode' background."
      ((,class (:foreground ,monokai-green))
       (,terminal-class (:foreground ,terminal-monokai-green))))
 
+   ;; which-key
+   `(which-key-key-face
+     ((,class (:foreground ,monokai-green
+			   :weight bold))
+      (,terminal-class (:foreground ,terminal-monokai-green
+				    :weight bold))))
+
+   `(which-key-separator-face
+     ((,class (:foreground ,monokai-comments))
+      (,terminal-class (:foreground ,terminal-monokai-comments))))
+
+   `(which-key-note-face
+     ((,class (:foreground ,monokai-comments))
+      (,terminal-class (:foreground ,terminal-monokai-comments))))
+
+   `(which-key-command-description-face
+     ((,class (:foreground ,monokai-fg))
+      (,terminal-class (:foreground ,terminal-monokai-fg))))
+
+   `(which-key-local-map-description-face
+     ((,class (:foreground ,monokai-yellow-hc))
+      (,terminal-class (:foreground ,terminal-monokai-yellow-hc))))
```

Comment:
I find that with monokai-yellow it might make the distinction between command groups and commands less instinctive, and I find it smoother to the eye.

But then again, it is personnal taste and even if I find it still distinguishable it might not be the case for everyone or everyone's screen, so I might be wrong.

Here is a screenshot with monokai-yellow so that you can see for yourself:

![screenshot from 2016-06-28 11-07-38](https://cloud.githubusercontent.com/assets/6248219/16409986/95211daa-3d20-11e6-986a-f175aa02b152.png)